src/styles/app.scss

```diff
@@ -502,16 +523,15 @@ nav,
   color: $white;
   text-align: center;
   text-decoration: none;
-  font-weight: bold;
+  font-weight: 600;
   line-height: 1;
 }
 
 .nav-item--active,
-.nav-item:hover {
-  .nav-item-text {
+.text-selector:hover {
     color: $white;
+    text-decoration: underline;
```

Comment:
I think in the other places we're using "underlines" they're actually a bottom border.

You could try something like `border-bottom: 2px solid $white;` or similar

Reply:
Ah okay, that makes more sense. I can change it to that 👍